scripts/lib/mkcloud-libvirt.sh

```diff
@@ -413,6 +413,14 @@ function libvirt_do_setuplonelynodes()
         $sudo lvdisplay "$lonely_disk" || \
             _lvcreate "${cloud}.node$i" "${lonelynode_hdd_size}" "$cloudvg"
 
+        local j
+        local lonely_data_disk
+        for j in $(seq ${cephvolumenumber:=1}); do
```

Comment:
there is already a default set in mkcloud, so no need to have a default to 1 again here